src/pages/komodo-defi-framework/tutorials/query-the-mm2-database/index.mdx

```diff
@@ -3,7 +3,22 @@ export const description = "This guide describes how to query the MM2 SQLite dat
 
 # How to Query the MM2 SQLite Database
 
-The Komodo DeFi Framework API stores historical information such as swaps and orders within an SQLite database, located under the DB user data folder (e.g. {'{folder continaing mm2 binary}'}/DB/{'{wallet identifying hex string}'}/MM2.db').
+The Komodo DeFi Framework API stores historical information such as swaps and orders within an SQLite database, located under the DB user data folder with each wallet having its own subfolder represented by a hexideciaml string.
+This string is shown in the runtime logs of the `kdf` binary as `Public key hash` when it starts up.
+
+```bash
+30 08:39:23, mm2:269] AtomicDEX API 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, mm2_main::mm2::lp_native_dex:500] INFO Version: 2.1.0-beta_a81f2a101 DT 2024-07-26T23:24:52+03:00
+30 08:39:23, crypto::crypto_ctx:324] INFO Public key hash: 7d6cbdd91788df3b764247721fe12853ce36b03d
+```
+
+The location of this folder for each operating system is:
+
+```bash
+- Linux: $HOME/.kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
+- MacOS: $HOME/Library/Application Support/kdf/DB/{'{wallet identifying hex string}'}/MM2.db'
```

Comment:
On MacOs by default the DB path is the same as on Linux
![Screenshot 2024-09-13 at 18 13 25](https://github.com/user-attachments/assets/a31267f4-d62d-4bbe-b1e2-55cdb292d3bb)

Also please add doc description somewhere, that if you dont want DB to be generated in home/.kdf you need to provide path in `conf["dbdir"]` param

Reply:
fixed